_episodes_rmd/06-best-practices-R.Rmd

```diff
@@ -141,7 +141,7 @@ rm(list = ls()) # If you want to delete all the objects in the workspace and sta
 
 8. Collaborate. Grab a buddy and practice "code review". Review is used for preparing experiments and manuscripts; why not use it for code as well? Our code is also a major scientific achievement and the product of lots of hard work!
 
-9. Develop your code using version control and frequent updates! You can find lessons about version control on [software-carpentry.org/lessons][lessons].
+9. Develop your code using version control and frequent updates! You can find lessons about version control on [software-carpentry.org/lessons][swc-lessons].
```

Comment:
might worth trying: `[swc-lessons][swc-lessons]`.
might be helpful if `software-carpentry.org/lessons` moves to `carpentries.org/lessons`.

Reply:
Any hint, when this might happen? Will a redirect be set-up then? And wouldn't the `swc` part be changed then as well?

I think I understood your initial suggestion in #371 better now: quickly fixing the link, while the surroundings were still being set up, correct? I should have just merged it then m-)

Reply:
Right, but don't worry about that PR. I think carpentries/styles#303 makes perfect sense and should be merged soon.

Reply:
OK, then. We survived the broken link tus far ;-)